packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -1413,20 +1413,6 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
   /// [RawGestureDetector.gestures]. It acts like a temporary replacement for
   /// that value until the next build.
   void replaceGestureRecognizers(Map<Type, GestureRecognizerFactory> gestures) {
-    assert(() {
```

Comment:
This affects a whole lot more than the single child scroll view, why does this need to be removed for all widgets using gesture detectors?